src/ripple/overlay/impl/Message.cpp

```diff
@@ -17,15 +17,18 @@
 */
 //==============================================================================
 
-#include <ripple/basics/safe_cast.h>
 #include <ripple/overlay/Message.h>
 #include <ripple/overlay/impl/TrafficCount.h>
 #include <cstdint>
 
 namespace ripple {
 
+/** Number of bytes in a message header. */
+std::size_t constexpr headerBytes = 6;
```

Comment:
I'd put this variable in an anon namespace

Reply:
Done.